Makefile

```diff
@@ -1,19 +1,9 @@
 .DEFAULT_GOAL := help
 
-PYTHONPATH=
 SHELL=/bin/bash
 VENV=.venv
 VENV_BIN=$(VENV)/bin
-
-ifndef SCALE_FACTOR
-data/tables/.generated:
-	@echo "SCALE_FACTOR not set, skipping data table generation"
-	@touch $@
-
-data/tables/:
-	@echo "SCALE_FACTOR not set, skipping data table generation"
-	@mkdir -p $@
-endif
+PYTHON?=$(VENV_BIN)/python
```

Comment:
I had to pass `PYTHONPATH` generate the data and run the queries
1. `PYTHONPATH=. make data/tables/`
2. `PYTHONPATH=. make run-polars`
3. PYTHONPATH=. make run-polars-gpu-no-env
etc.

Reply:
Interesting, no. I actually find that I need to specify the PYTHONPATH unless I remove the line that I removed, so there is something strange going on here. Let me retest in a couple of different environments (and maybe ping you as well). It may be that we need to do something a bit more clever here to ensure that we support different environments or use cases.